File: Iniciante/URI 2930.go
package main
 
import (
    "fmt"
)
 
func main() {
    var e, d int64

    fmt.Scanf("%d", &e)
    fmt.Scanf("%d", &d)

	if (e > d) {
		fmt.Println("Eu odeio a professora!")
	} else if ((d - e) >= 3) {
		fmt.Println("Muito bem! Apresenta antes do Natal!")
	} else if ((e + 2) < 24) {
		fmt.Println("Parece o trabalho do meu filho!\nTCC Apresentado!")
	} else {
		fmt.Println("Parece o trabalho do meu filho!\nFail! Entao eh nataaaaal!")
	}
}

File: Iniciante/URI 2930.ml
let e, d = Scanf.scanf "%i %i" (fun a b -> a, b);;
if (e > d) then (
    Printf.printf "Eu odeio a professora!\n";
) else if ((d - e) >= 3) then (
    Printf.printf "Muito bem! Apresenta antes do Natal!\n";
) else if ((e + 2) < 24) then (
    Printf.printf "Parece o trabalho do meu filho!\nTCC Apresentado!\n";
) else (
    Printf.printf "Parece o trabalho do meu filho!\nFail! Entao eh nataaaaal!\n";
)